src/components/EventForm/EventForm.js

```diff
@@ -203,6 +203,14 @@ function EventForm({ courses, friends, hostId, refreshEvents }) {
               {isPrivate && (
                 <>
                   <p className='friends-title'>Friends to Invite:</p>
+                  { !friends.length && 
+                    <div>
+                      <em className='friendless'>
+                        You don't have any friends...<br/>
+                        Make some by creating a public event!
+                      </em>
+                    </div>
+                  }
```

Comment:
Good thinking adding this error handling! Suggesting they create a public event is a nice touch!